compiler/AST/expr.cpp

```diff
@@ -3786,7 +3786,8 @@ void CallExpr::prettyPrint(std::ostream *o) {
         argList.last()->prettyPrint(o);
         *o << ")";
       } else if (strcmp(expr->unresolved, "chpl_build_bounded_range") == 0 ||
-                 strcmp(expr->unresolved, "chpl_build_partially_bounded_range") == 0 ||
+                 strcmp(expr->unresolved, "chpl_build_low_bounded_range") == 0 ||
+                 strcmp(expr->unresolved, "chpl_build_high_bounded_range") == 0 ||
                  strcmp(expr->unresolved, "chpl_build_unbounded_range") == 0) {
         argList.first()->prettyPrint(o);
         *o << "..";
```

Comment:
I may not have looked very carefully at this last time, but I'm not seeing what about this logic would cause a low bounded range to be pretty-printed as 'lo..' and a high-bounded range to be printed '..hi'.